ledger/catchupaccessor.go

```diff
@@ -800,7 +806,7 @@ func (c *CatchpointCatchupAccessorImpl) finishBalances(ctx context.Context) (err
 	start := time.Now()
 	ledgerCatchpointFinishBalsCount.Inc(nil)
 	err = wdb.Atomic(func(ctx context.Context, tx *sql.Tx) (err error) {
-		var balancesRound uint64
+		var balancesRound, hashRound uint64
```

Comment:
not sure I understand why balancesRound and hashRound can be different, looks like we were updating them simultaneously.

Reply:
In some cases, we want to reset the hashRound by setting it to zero.